lib/builder.js

```diff
@@ -111,7 +112,8 @@ Builder.prototype.build = function (willReadStringTree) {
       name: getDescription(tree),
       broccoliNode: true,
       broccoliId: node.id,
-      broccoliCachedNode: false
+      broccoliCachedNode: false,
+      broccoliPluginName: getPluginName(tree)
```

Comment:
I would also be happy with the following (though assume it were better factored to avoid duplicate work):

```js
    heimdall.start({
      name: getPluginName + ': ' + getDescription(tree),
      broccoliNode: true,
      broccoliId: node.id,
      broccoliCachedNode: false,
      broccoliPluginName: getPluginName(tree),
      broccoliAnnotation: getDescription(tree)
    });
```